diesel_tests/tests/joins.rs

```diff
@@ -678,13 +678,13 @@ fn connection_with_fixture_data_for_multitable_joins() -> (TestConnection, TestD
         .order(posts::id)
         .load::<Post>(&connection)
         .unwrap();
-    let new_comments = vec![
+    let new_comments: &[NewComment<'static>] = &[
         NewComment(posts[0].id, "First Comment"),
```

Comment:
Any idea why this fails on nightly? Should we maybe report a regression against rustc?

Reply:
I have no idea. We haven't changed behavior for it and the failure only appears on nightly so I guess recent changes on nightly cause the failure (I'm not sure it's a regression or expected behavior though).

Reply:
I've retriggered the ci. Seems to be fixed now. You need to run rustfmt.